lookup/simple_env.go

```diff
@@ -3,14 +3,12 @@ package lookup
 import (
 	"fmt"
 	"os"
-
-	"github.com/docker/libcompose/project"
 )
 
 type OsEnvLookup struct {
 }
 
-func (o *OsEnvLookup) Lookup(key, serviceName string, config *project.ServiceConfig) []string {
+func (o *OsEnvLookup) Lookup(key string) []string {
```

Comment:
Simplified the method signature as serviceName and project.ServiceConfig are not needed at all there. I *reported* that to `EnvironmentLookup` too.